lib/evm/eei.ts

```diff
@@ -36,7 +54,7 @@ export default class EEI {
   }
 
   refundGas (amount: BN): void {
-    this._result.gasRefund.iaddn(amount)
+    this._result.gasRefund.iadd(amount)
   }
```

Comment:
Uh, this didn't cause a bug before?

Reply:
It wasn't a bug because `refundGas` was called with a number always (see changes in `opFns`). I changed them to pass a `BN` instead of a normal number.